libs/design-system-components/src/components/Editor/index.tsx

```diff
@@ -448,6 +448,8 @@ const EditorBox: React.FC<EditorBoxProps> = props => {
 
   const publishDisabled = publishDisabledInternal || disablePublish;
 
+  const [, forceUpdate] = React.useReducer(x => x + 1, 0);
```

Comment:
why is a forced update needed?

Reply:
because when formatting text slate doesn't update the children, the force update is for the changes to propagate to the children so the content block has the latest state of the editor. And since this happens only when formatting text it is not very intensive, as it happens infrequently